package.json

```
@@ -173,7 +176,7 @@
   },
   "resolutions": {
     "eslint-config-ckeditor5": "^4.0.0",
-    "postcss-loader": "^4.0.0",
+    "postcss-loader": "^4.3.0",
```

```suggestion
    "postcss-loader": "^4.0.0",
```

The previous and proposed change will install the same package.